README.md

```diff
@@ -86,7 +86,9 @@ To run on `regtest` you have to change the default network (`testnet`) to `regte
 
 Example command for maker:
 
-`NETWORK=regtest make maker`
+`make maker`
+
+(`make` runs prepend NETWORK=regtest automatically)
 
 Example command for taker:
```

Comment:
Maybe change the command to `make run`? 
(above as well...)

Reply:
like `make run maker`, `make run taker`?

Reply:
the commands I changed are not just for running, they build too, hence I'm hesitant to change to *run*.